ballerina/Ballerina.toml

```diff
@@ -16,3 +17,21 @@ groupId = "io.ballerina.lib"
 artifactId = "jsondata-native"
 version = "0.1.0"
 path = "../native/build/libs/data.jsondata-native-0.1.0-SNAPSHOT.jar"
+
+[[platform.java11.dependency]]
```

Comment:
```suggestion
[[platform.java17.dependency]]
```

Reply:
We cannot do this, this is only support for java 11